tracex_project/patient_journey_generator/templates/journey_generator_overview.html

```diff
@@ -8,37 +8,44 @@
     <script src="{% static '/patient_journey_generator/js/loading-spinner.js' %}" defer></script>
 </head>
 
-<body class="selection_body">
+<body class="main_body">
```

Comment:
This change leads to 
![grafik](https://github.com/bptlab/TracEX/assets/116206992/11dd28d3-455e-4f5a-8aea-5e8d1a7ef63f)
I would suggest to revert it.

Reply:
your browser is using the style.css from ur cache thats why the design is off. I will resolve this thread for now, please reopen if u need assistance